### DISCUSSION TITLE
自建一个官网

### DISCUSSION THREAD

**Discussion by Wu-jiyan** [LTCatt/PCL2Help#290] [Ideas]
好像大部分启动器都有自己的官网，但PCL2没有，我认为这样不方便小白查找PCL并下载，可以利用Github/Gitee Pages/Vercel自建一个官网，对小白更加友好，否则很容易会找不到PCL。
纯属个人提议，勿喷，谢谢！

**Comment by WForst-Breeze** [comment]
这里是 PCL2 帮助库（）而且龙猫明确说过不会做官网🤔
